### DISCUSSION TITLE
seemly innocuous comma breaks alias deserialization.

### DISCUSSION THREAD

**Discussion by wenhoujx** [pydantic/pydantic#4135] [Question] [answered]
```
from pydantic import Field, BaseModel


class Foo(BaseModel):
    schema_: str = Field(alias='schema'), # note the comma here!!!!!!!!!!
    account: str


if __name__ == '__main__':
    js = {'schema': 'public', 'account': 'test-account'}
    foo1 = Foo(**js)
    print(foo1)
```
output 
```
schema_=(FieldInfo(default=PydanticUndefined, alias='schema', alias_priority=2, extra={}),) account='test-account'
```

with out the `,` (comma) at the end of `schema_` field , the field is populated correctly with `public`, can someone educate me why is it the case? Thanks

**Comment by samuelcolvin** [comment, marked as answer]
Well it's now a tuple, this is an easy mistake, but it's true throughout python.

Nothing we can do.

**Comment by wenhoujx** [comment]
@samuelcolvin  thanks for the education, i suspected this was b/c python not pydantic, by no means trying to bug report this, I was trying to understand it better. 

Thanks !
